doc/source/schemas/alleleAnnotations.rst

```diff
@@ -141,14 +141,14 @@ This protocol defines types used by the GA4GH Allele Annotation API.
   :field description:
     A description of the experiment.
   :type description: null|string
-  :field created:
+  :field createTime:
```

Comment:
@diekhans Time definitions are in the api -> metadata.rst / html file; in my biosample branch and derivatives. A rendered doc is here:
* http://ga4ghdata.org/Sites/github/mbaudis/biosample/html/api/metadata.html